src/js/rx/components/ConfirmRefillModal.jsx

```diff
@@ -13,11 +14,11 @@ class ConfirmRefillModal extends React.Component {
     // and possibly a server request.
     event.preventDefault();
     const alertContent = (
-      <span>
-        Refill for {this.props.drug} has been requested.
-      </span>
+      <b>
```

Comment:
If alert content is generally going to be bold, I'd rather see it set in the stylesheet. Maybe for all `.rx-alert-text`. Also, just noticed the partial is still named `_va-alerts`, but can probably be renamed to `_rx-alert`.